monitor/src/main/java/edu/asu/diging/monitor/web/admin/AddRecipientController.java

```diff
@@ -22,10 +22,11 @@
 	@Autowired
 	private INotificationManager manager;
 	
+	
 	@RequestMapping(value="/admin/recipients/add", method=RequestMethod.GET)
 	public String show(Model model) {
 		model.addAttribute("recipientForm", new RecipientForm());
-		return "admin/recipients/show";
+		return "admin/recipients/add";
```

Comment:
for the below method: after a new user has been added, the list page should be shone.